keyboards/handwired/dactyl_left/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "dactyl_left", 
+    "url": "", 
+    "maintainer": "RedForty", 
+    "width": 6.5, 
+    "height": 7.375, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [{"x":3.5, "y":0}, {"x":2.5, "y":0.125}, {"x":4.5, "y":0.125}, {"x":5.5, "y":0.25}, {"x":0, "y":0.375, "w":1.5}, {"x":1.5, "y":0.375}, {"x":3.5, "y":1}, {"x":2.5, "y":1.125}, {"x":4.5, "y":1.125}, {"x":5.5, "y":1.25}, {"x":0, "y":1.375, "w":1.5}, {"x":1.5, "y":1.375}, {"x":3.5, "y":2}, {"x":2.5, "y":2.125}, {"x":4.5, "y":2.125}, {"x":5.5, "y":2.25}, {"x":0, "y":2.375, "w":1.5}, {"x":1.5, "y":2.375}, {"x":3.5, "y":3}, {"x":2.5, "y":3.125}, {"x":4.5, "y":3.125}, {"x":5.5, "y":3.25}, {"x":0, "y":3.375, "w":1.5}, {"x":1.5, "y":3.375}, {"x":3.5, "y":4}, {"x":2.5, "y":4.125}, {"x":4.5, "y":4.125}, {"x":0.5, "y":4.375}, {"x":1.5, "y":4.375}, {"x":1, "y":4.375}, {"x":2, "y":4.375}, {"x":0, "y":5.375, "h":2}, {"x":1, "y":5.375, "h":2}, {"x":2, "y":5.375}, {"x":2, "y":6.375}]
```

The vertical stagger of the columns means QMK's KLE to info.json converter doesn't order the key objects correctly (it counts `"y":0.125` as a different row from `"y":0`).

```suggestion
            "layout": [{"x":0, "y":0.375, "w":1.5}, {"x":1.5, "y":0.375}, {"x":2.5, "y":0.125}, {"x":3.5, "y":0}, {"x":4.5, "y":0.125}, {"x":5.5, "y":0.25}, {"x":0, "y":1.375, "w":1.5}, {"x":1.5, "y":1.375}, {"x":2.5, "y":1.125}, {"x":3.5, "y":1}, {"x":4.5, "y":1.125}, {"x":5.5, "y":1.25}, {"x":0, "y":2.375, "w":1.5}, {"x":1.5, "y":2.375}, {"x":2.5, "y":2.125}, {"x":3.5, "y":2}, {"x":4.5, "y":2.125}, {"x":5.5, "y":2.25}, {"x":0, "y":3.375, "w":1.5}, {"x":1.5, "y":3.375}, {"x":2.5, "y":3.125}, {"x":3.5, "y":3}, {"x":4.5, "y":3.125}, {"x":5.5, "y":3.25}, {"x":0.5, "y":4.375}, {"x":1.5, "y":4.375}, {"x":2.5, "y":4.125}, {"x":3.5, "y":4}, {"x":4.5, "y":4.125}, {"x":6, "y":4.75}, {"x":7, "y":4.75}, {"x":5, "y":5.75, "h":2}, {"x":6, "y":5.75, "h":2}, {"x":7, "y":5.75}, {"x":7, "y":6.75}]
```